core/common/assets/js/api/core/hooks/base.js

```diff
@@ -69,6 +69,10 @@ export default class HooksBase extends elementorModules.Module {
 		elementorModules.forceMethodImplementation();
 	}
 
+	get( id ) {
```

Comment:
You can use an object to store the `callbacksFlatList` with callback id as a key, then you don't need to walk through all callbacks to find the desired one

Reply:
Handled: https://github.com/elementor/elementor/pull/12097/commits/c8e075954e693018ff59a1d58e6e5d4153ea3319